apps/functions-subscription/src/domain/trial.ts

```diff
@@ -35,13 +40,36 @@ export interface TrialWriter {
   ) => TE.TaskEither<Error | ItemAlreadyExists, Trial>;
 }
 
+export interface TrialReader {
+  readonly get: (trialId: TrialId) => TE.TaskEither<Error, O.Option<Trial>>;
+}
+
 export const makeTrial = (
```

Comment:
**chore**: I'm not sure, but I think we can remove the `export` keyword for the `makeTrial` function